lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -342,6 +348,10 @@ <h4 class="message-title">${_('Your verification will expire soon!')}</h4>
           </div>
         %endif
 
+        % if course_program_info:
+            <%include file = "_dashboard_xseries_info.html" args="course_program_info=course_program_info, program_marketing_url=program_marketing_url,enrollment_mode=enrollment.mode" />
```

Comment:
...and here, could you add `and course_program_info['category']=='xseries'` to the conditional?